nimsuggest/tests/tv3_con.nim

```diff
@@ -0,0 +1,13 @@
+# tests v3
+
+proc test(a: string, b:string) = discard
+proc test(a: int) = discard
+
+test(#[!]#
+
+discard """
+$nimsuggest --v3 --tester $file
+>con $1
+con     skProc  tv3_con.test    proc (a: string, b: string)     $file      3       5       ""      100
+con     skProc  tv3_con.test    proc (a: int)    $file    4       5       ""      100
```

Comment:
```suggestion
con;;skProc;;tv3_con.test;;proc (a: string, b: string);;$file;;3;;5;;"";;100
con;;skProc;;tv3_con.test;;proc (a: int);;$file;;4;;5;;"";;100
```
`nimsuggest/tester.nim` will replace `;;` with tabs. Saves needing to worry about the formatting

Reply:
Thanks, wasnt able to run in locally